README.md

```diff
@@ -36,6 +36,8 @@ This repository is broken up into:
     *   [query_audit](/views/audit/query_audit.sql) - View to simplify querying
         the audit logs which can be used to power dashboards
         ([example](https://codelabs.developers.google.com/codelabs/bigquery-pricing-workshop/#0)).
+*   [third_party](/third_party) - Relevant third party libraries for BigQuery 
```

Comment:
Add this before `Tools` to keep in alphabetical order and rename to `Third Party`

Reply:
Done.